spring-security-rest-full/.springBeans

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beansProjectDescription>
 	<version>1</version>
-	<pluginVersion><![CDATA[3.8.2.201610040608-RELEASE]]></pluginVersion>
+	<pluginVersion><![CDATA[3.6.3.201411271034-RELEASE]]></pluginVersion>
```

Comment:
Why are we changing to the 2014 version?

Reply:
This is auto generated file, I have no idea why it changed